src/Sulu/Bundle/ContactBundle/DataFixtures/ORM/LoadDefaultTypes.php

```diff
@@ -34,17 +34,17 @@ public function load(ObjectManager $manager)
 
         $phoneType1 = new PhoneType();
         $phoneType1->setId(1);
-        $phoneType1 = $manager->merge($phoneType1);
+        $manager->persist($phoneType1);
```

Comment:
A check need to be added to check if the fixture already exists. https://github.com/sulu/sulu/issues/7184#issue-1952697265.

Else it will fail if when `sulu:build dev` is run multiple times which should still be allowed.

Reply:
Because of the hardcoded id, you mean?

Reply:
Yes and that the fixtures not getting duplicated.

Reply:
So the logic would need to be changed like this?
```php
$phoneType1 = $manager->find(PhoneType::class, 1);
if ($phoneType1 === null) {
     $phoneType1 = new PhoneType();
     $phoneType1->setId(1);
     $manager->persist($phoneType1);
}
$phoneType1->setName('sulu_contact.work');
```

Reply:
all can be inside the if statement.